modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -0,0 +1,240 @@
+* *Method Label Options Color*: Set the color of the labels of *OPTIONS* methods.
+* *Method Label Head Background Color*: Set the background color of the labels of *HEAD* methods.
+* *Method Label Head Color*: Set the color of the labels of *HEAD* methods.
+* *Link Hover Color*: Set the color of links when the cursor hovers over them.
+* *Button Text Color (Optional)*: Set the color of text on buttons.
+* *Button Hover Text Color (Optional)*: Set the color of text on buttons when the cursor hovers over them. Default is the button text color.
+* *Headline Font Family (Optional)*: Set the font family of the headline.
+* *Error Color (Optional)*: Set the color of error messages.
+
+== API Documentation Viewer
+
+The API documentation viewer component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
+
+* *API*: Choose the API.
+* *Section (Optional)*: Set the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
+* *Selected Page (Optional)*: Set the page to open.
+
+== API Header
+
+The API header component displays the API Header including icon, name, and description.
+
+* *API*: Choose the API to show.
+
+== Application Analytics Graph
+
+The application analytics graph component displays statistics about the use of an application.
+
+* *Application*: Choose the application to show.
+
+== Application Details
+
+The application details component displays details about an application.
+
+// check component description
+
+* *Application*: Choose the API to show.
+
+== Application Listing
+
+The application listing component shows applications that use an API.
+
+* *New Application URL*: Set a new URL for the application.
+* *Card Margin*: Set the amount of margin space around the card.
+* *Card Border*: Set the thickness of the card borders.
+* *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
+* *Card Shadow*: Set the shadow beneath the card.
+* *Card Width*: Set the card width.
+* *Card Height*: Set the card height.
+
+// check *New Application URL* description
+
+== Login Form
+
+The login form component allows members to log in to the portal.
+
+* *Start URL*: Set the URL members see after completing login. Default is the portal home page.
+* *Username Label*: Set the label for the username field. Default is `Email`.
+* *Password Label*: Set the label for the password field. Default is `Password`.
+* *Login Button Label*: Set the label for the login button. Default is `Log in`.
+* *Forgot Password Label*: Set the label for the forgotten password link. Default is `Forgot your password?`.
+* *Forgot Password URL*: Set the URL for the forgotten password link. Default is `/ForgotPassword`.
+* *Self Register Label*: Set the label for the self registration link. Default is `Not a member?`.
```

Comment:
Here and below, "self registration link" > "self-registration link"

Reply:
Fixed in all instances.